integration_tests/seeds/hubspot_engagement.csv

```diff
@@ -0,0 +1,2 @@
+id,type,_fivetran_synced,portal_id
```

Comment:
Is there a reason only a subset of seed columns are being run and tested here? Can we add all the relevant columns brought into the staging model?

Reply:
that's a great point.  i initially mocked this up from the internal schema with the new version of tables that i was testing with. I added in the rest of the columns!